services-js/registry-certs/client/birth/ReviewRequest.tsx

```diff
@@ -0,0 +1,223 @@
+              </p>
+              <p>
+                If you are requesting a restricted record, you will need to
+                provide a valid form of identification (i.e. driver’s license,
+                state ID, military ID, or passport) after you pay.
+              </p>
+
+              <div>
+                <div className={CERTIFICATE_ROW_STYLE}>
+                  <input
+                    type="number"
+                    min="1"
+                    aria-label="Certificate quantity"
+                    className={`br br-a150 ${QUANTITY_BOX_STYLE}`}
+                    value={this.state.certificateQuantity}
+                    onChange={this.handleQuantityChange}
+                    title="Certificate Quantity"
+                  />
+
+                  <div className={`t--sans ${CERTIFICATE_INFO_BOX_STYLE}`}>
+                    <div className={CERTIFICATE_NAME_STYLE}>
+                      Birth Certificate (Paper Copy)
+                    </div>
+                    <div className={CERTIFICATE_SUBINFO_STYLE}>
+                      <span style={{ marginRight: '2em' }}>
+                        {firstName} {lastName}
+                      </span>
+                      <span>Born: {birthDate}</span>
+                    </div>
+                  </div>
+
+                  <button
+                    className={REMOVE_BUTTON_STYLE}
+                    type="button"
+                    onClick={this.userResetStartOver}
+                    aria-label="Remove and start over"
+                    title="Remove and start over"
+                  >
+                    ×
+                  </button>
+                </div>
+              </div>
+
+              <CostSummary
+                certificateType="birth"
+                certificateQuantity={this.state.certificateQuantity}
+                allowServiceFeeTypeChoice
+                serviceFeeType="CREDIT"
+              />
+
+              <div className={BUTTONS_CONTAINER_STYLING}>
+                <button
+                  className={`btn ${SECONDARY_BUTTON_STYLE}`}
+                  type="button"
+                  onClick={this.returnToQuestions}
+                >
+                  Back to last question
+                </button>
+
+                <button
+                  className="btn"
+                  type="button"
+                  onClick={this.goToCheckout}
+                >
+                  Proceed to checkout
+                </button>
+              </div>
+            </section>
+          </div>
+        </PageLayout>
+      </>
+    );
+  }
+}
+
+const QUANTITY_BOX_STYLE = css({
+  width: '2.5rem',
+  height: '2.5rem',
+  marginRight: '1rem',
+  fontFamily: 'inherit',
+  fontStyle: 'italic',
+  fontSize: '1rem',
+  background: OPTIMISTIC_BLUE,
+  color: 'white',
+  textAlign: 'right',
+  padding: '0.5rem',
+  '-moz-appearance': 'textfield',
+  '&::-webkit-inner-spin-button': {
```

Comment:
Is this the invocation to get rid of the little up/down arrows? TIL.

Reply:
Yes ma’am! I have been meaning to write a post on Medium or something that lists all of them (i.e. these are the pseudo elements in a range input, there are the ones in a number input) in one spot...